packages/expo-cli/src/commands/start.js

```diff
@@ -27,6 +28,17 @@ async function action(projectDir, options) {
   if (options.clear) {
     startOpts.reset = true;
   }
+  if (options.https) {
+    startOpts.https = true;
+  }
+  if (options.host) {
+    if (typeof options.host === 'string') {
+      startOpts.host = options.host;
+    } else {
+      startOpts.host = ip.address();
```

Comment:
We also accept `EXPO_PACKAGER_HOSTNAME` and `REACT_NATIVE_PACKAGER_HOSTNAME`: https://github.com/expo/expo-cli/blob/6dc22d9cf7796340293f8844515b0fa538147572/packages/xdl/src/UrlUtils.js#L238-L247.

In order to not make things any more complicated, could we just reuse that logic in `constructWebAppUrlAsync` and not have a `--host` flag at all? (You can extract that bit out to a separate function.)

The `--host` flag might be a bit confusing because it only applies to the webpack dev server and not other URLs created by `expo start`. And I would rather not have three or four ways to set the hostname.

Reply:
we should do this in another PR. I'll move the current IP related changes.